templates/base.html.twig

```diff
@@ -69,38 +61,6 @@
     </div>
 </div>
 <div class="page-container">
-
-    {% if current_route == 'app_identity_registration' %}
-        <div class='alert'>
-            <div><i class="fa fa-info-circle" aria-hidden="true"></i>{{ 'page.route.description.documentation_purpose'|trans }}</div>
-            <pre>{{ 'page.route.description.app_identity_registration'|trans }}</pre>
-        </div>
-    {% endif %}
-    {% if current_route == 'app_identity_authentication' %}
-        <div class='alert'>
-            <div><i class="fa fa-info-circle" aria-hidden="true"></i>{{ 'page.route.description.documentation_purpose'|trans }}</div>
-            <pre>{{ 'page.route.description.app_identity_authentication'|trans }}</pre>
-        </div>
-    {% endif %}
-    {% if current_route == 'gssp_saml_sso' %}
-        <div class='alert'>
-            <div><i class="fa fa-info-circle" aria-hidden="true"></i>{{ 'page.route.description.documentation_purpose'|trans }}</div>
-            <pre>{{ 'page.route.description.gssp_saml_sso'|trans }}</pre>
-        </div>
-    {% endif %}
-    {% if current_route == 'gssp_saml_sso_return' %}
-        <div class='alert'>
-            <div><i class="fa fa-info-circle" aria-hidden="true"></i>{{ 'page.route.description.documentation_purpose'|trans }}</div>
-            <pre>{{ 'page.route.description.gssp_saml_sso_return'|trans }}</pre>
-        </div>
-    {% endif %}
-    {% if current_route == 'sp_demo' %}
-        <div class='alert'>
-            <i class="fa fa-info-circle" aria-hidden="true"></i>
-            <pre style="display: inline">{{ 'page.route.description.sp_demo'|trans }}</pre>
-        </div>
-    {% endif %}
```

Comment:
I removed these blocks in the Azure MFA project, but we might want to keep them in the example project..

Reply:
:+1: will restore it